journal/original/01-journal.adoc

```diff
@@ -1517,9 +1517,9 @@ he keeps his hat so fast upon the block.
 Then he intending and preparing to come down a great steep ditch,
 I stept down to lend him my hand to help him;
 another priest would have stept between me and him,
-but Lord Herbert refused the priest`'s help; and stopping a little, said to the priest.
-Here is a brother that stands by will say.
-The blind leads the blind, and both will fall into the ditch.
+but Lord Herbert refused the priest`'s help; and stopping a little, said to the priest,
+`'Here is a brother that stands by will say,
+The blind leads the blind, and both will fall into the ditch.`'
```

Comment:
It's always a tough judgment call on how much to do to the original editions, but here I kind of lean towards doing the same punctuation in the original as we do in the modernized, with the outer and inner quotations marks on this sentence.